e2etest/utils.go

```diff
@@ -76,3 +136,31 @@ func tweakRCPort() func(rc *v1.ReplicationController) {
 	}
 	return nil
 }
+
+func updateConfigMapObjs(cs clientset.Interface, dataModifier func(m *configFile)) error {
+	cm, err := cs.CoreV1().ConfigMaps(testNameSpace).Get(context.TODO(), "config", metav1.GetOptions{})
+	if err != nil {
+		return pkgerrors.Wrapf(err, "Failed to get MetalLB ConfigMap")
+	}
+
+	if len(cm.Data) == 0 {
+		cm.Data = make(map[string]string)
```

Comment:
Do you need to do this? Here we are always overriding the whole config right?

Reply:
Also, wouldn't everything be simpler if we always delete and create the configmap with the new config? Am I missing something?